pkg/controller/elasticsearch/reconcile/status.go

```diff
@@ -240,7 +240,7 @@ func (d *DownscaleReporter) RecordNodesToBeRemoved(nodes []string) {
 // Merge creates a new downscale status using the reported downscale status and an existing downscale status.
 func (d *DownscaleReporter) Merge(other esv1.DownscaleOperation) esv1.DownscaleOperation {
 	downscaleOperation := other.DeepCopy()
-	if d == nil {
+	if d == nil || (len(d.nodes)+len(other.Nodes) == 0) {
```

Comment:
I'm wondering if there are some cases where the `LastUpdatedTime` might not been updated if the value of `stalled` is updated.